include/seqan3/io/alignment_file/detail.hpp

```diff
@@ -46,15 +47,12 @@ struct view_equality_fn
  * \param  reference_char      The aligned character of the reference to compare.
  * \param  query_char          The aligned character of the query to compare.
  * \param  extended_cigar      Whether to print the extended cigar alphabet or not. See CIGAR operation.
- * \returns A char representing the alignment operation between the
- *          two values.
+ * \returns A seqan3::cigar_op representing the alignment operation between the two values.
  *
  * \details
  *
- * \attention Note that CIGAR elements (respectively by their
- *            CIGAR operation) are always related to on one of the two sequences
- *            in a pairwise alignment. In this case, the resulting
- *            operation is related to the \p query_char.
+ * \attention Note that CIGAR elements (respectively by their CIGAR operation) are always related to one of the two
+ *            sequences in a pairwise alignment. In this case, the resulting operation is related to the \p query_char.
```

Comment:
you might want to grep for \p and then replace it with tick marks as the desired behavior seems not to work.